.github/workflows/apt.yml

```diff
@@ -46,6 +46,12 @@ jobs:
       - name: Build deb with Docker
         run: |
           rake apt:build APT_TARGETS=${{ matrix.rake-job }}
+      - name: Build v6 deb with Docker
+        run: |
+          git config user.email "fluentd@googlegroups.com"
+          git config user.name "Fluentd developers"
+          git am fluent-package/bump-version-v6.patch
+          rake apt:build APT_TARGETS=${{ matrix.rake-job }}
```

Comment:
Can we build this dummy v6 package into a separate path?
This dummy v6 package would be very helpful for CI, but it would be better to avoid impact on the existing tests because it would be a temporary process.

Reply:
Use name: in upload-artifacts.